src/Plugin/GraphQL/Fields/FieldPluginBase.php

```diff
@@ -118,16 +117,14 @@ public function resolve($value, array $args, ResolveContext $context, ResolveInf
       if (array_key_exists($argument, $args) && !is_null($args[$argument])) {
         $context->setContext($argument, $args[$argument], $info);
       }
-      else {
-        $args[$argument] = $context->getContext($argument, $info);
-      }
+      $args[$argument] = $context->getContext($argument, $info, $context->getGlobal($argument));
```

Comment:
Do we need something else than getGlobal / setGlobal? Basically set the root field? Globals were rather meant as a configuration value. Instead of having to use getContext() with a default value everytime, I'd rather set a root context so getContext() automatically finds it.